techniques/reflectometry/__init__.py

```diff
@@ -0,0 +1,5 @@
+
+import base
+import sample
```

Comment:
I'd change this to

from . import base, sample

This will future proof us for Python 3.